src/Raven.Server/Smuggler/Documents/Data/ISmugglerDestination.cs

```diff
@@ -111,6 +111,10 @@ public interface ICompareExchangeActions : INewCompareExchangeActions, IAsyncDis
         ValueTask WriteKeyValueAsync(string key, BlittableJsonReaderObject value, Document existingDocument);
 
         ValueTask WriteTombstoneKeyAsync(string key);
+
+        bool MaybeFlush();
+
+        Task FlushAsync();
```

Comment:
ValueTask?

Reply:
Explicitly no, since we'll _always_ wait here.

Reply:
then just await the SendClusterTransactionsAsync instead of converting it to the Task?

Reply:
Isn't that the same thing?

Reply:
My OCD kicks in, sorry :)

Reply:
Done